src/app/components/Ad/Amp/AdSlot/index.test.tsx

```diff
@@ -20,20 +20,29 @@ describe('getAssetType', () => {
 
 describe('Amp AdSlot', () => {
   describe('Snapshots', () => {
-    shouldMatchSnapshot(
-      'should correctly render a leaderboard AdSlot',
-      <AdSlot service="mundo" slotType="leaderboard" pageType="STY" />,
-    );
-    shouldMatchSnapshot(
-      'should correctly render a mpu AdSlot',
-      <AdSlot service="mundo" slotType="mpu" pageType="IDX" />,
-    );
+    it('should correctly render a leaderboard AdSlot', () => {
+      const { container } = render(
+        <AdSlot service="mundo" slotType="leaderboard" pageType="STY" />,
+        { service: 'mundo' },
+      );
+      expect(container).toMatchSnapshot();
+    });
+
+    it('should correctly render a mpu AdSlot', () => {
+      const { container } = render(
+        <AdSlot service="mundo" slotType="mpu" pageType="IDX" />,
+        { service: 'mundo' },
+      );
+      expect(container).toMatchSnapshot();
+    });
   });
 });
 
 describe('getDataSlot', () => {
+  const originalAppEnv = process.env.SIMORGH_APP_ENV;
+
   afterEach(() => {
-    delete process.env.SIMORGH_APP_ENV;
+    process.env.SIMORGH_APP_ENV = originalAppEnv;
```

Comment:
Should this have been left as it was  - i.e. keep the `delete process.env.SIMORGH_APP_ENV;` 🤔